src/i_video.c

```diff
@@ -739,14 +739,11 @@ void I_GetEvent(void)
                 break;
 
             case SDL_QUIT:
-/*
```

Comment:
ACtually, I have left that code in there for the very specific reason to implement this feature, but somehow forgot about it since.